### ISSUE TITLE
Kotlin Gradle plugin version error

### ISSUE THREAD

**Issue by mdr** [michael-cheung-thebus/foreground_service#6]
When I added `foreground_service` as a dependency, I started getting an error:

```
FAILURE: Build failed with an exception.

* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.3.0 and higher.
The following dependencies do not satisfy the required version:
project ':foreground_service' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.71
```

Would it be possible to upgrade the version of `kotlin-gradle-plugin` used?

**Comment by Nicola-Cestaro** [comment]
Yeah I got the same Issue

**Comment by jjvillavicencio** [comment]
Any solution?

**Comment by Aizen741** [comment]
let me work on this issue..! :)

**Comment by michael-cheung-thebus** [comment]
Published v0.3.1 for this.
